### PR TITLE
[Blog Post] Cats Ecosystem community survey 2018 results

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

**Comment by kailuowang** [comment]
Sure, how about under /img/blog/?

**Comment by larsrh** [comment]
I propose `/img/media/2018-survey`.

**Comment by kailuowang** [comment]
@larsrh done.

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+title: Cats Ecosystem Community Survey 2018 Results
+
+meta:
+  nav: blog
+  author: kailuowang
+  pygments: true
+
+---
+
+Overall we received 588 responses over the course of 30 days. These feedbacks are essential for us to make informed decisions on our 2019 plan. Thank you, everyone, who participated.
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: `This feedback is`

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+meta:
+  nav: blog
+  author: kailuowang
+  pygments: true
+
+---
+
+Overall we received 588 responses over the course of 30 days. These feedbacks are essential for us to make informed decisions on our 2019 plan. Thank you, everyone, who participated. 
+
+As promised, here are the results, as well as some quick read from us.
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: `quick reads`

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+46% of the respondents are relatively new (less than 1 year) users. Welcome!
+
+<br /><br /><br />
+### Q: Do you feel welcome in the Cats ecosystem community?
+
+#### Results:
+![2](/img/media/2018-survey/JjK3muU.png)
+
+#### Our read:
+83% of users gave the 4+ ratings on the welcomeness of the community. This is a promising sign. We should aim for having more 5 ratings (currently at 42.5%) and less 3- ones.
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: s/`less`/`fewer`/

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+![2](/img/media/2018-survey/JjK3muU.png)
+
+#### Our read:
+83% of users gave the 4+ ratings on the welcomeness of the community. This is a promising sign. We should aim for having more 5 ratings (currently at 42.5%) and less 3- ones. 
+
+<br /><br /><br />
+### Q: How can we be more welcoming?
+
+#### Responses summary: 
+We received 92 responses on this free form question. The vast majority, 90+%, of them suggested that more documentation, tutorial, and real-world examples will help them feel more welcome. Some of them suggested that the learning curve could be less intimidating to them if there are more introductory resources.
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: `free-form`
`tutorials`
`would help them feel`
`if there were more`

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+![5](/img/media/2018-survey/3ne70PU.png)
+
+The free form `other` responses can be summarized as 
+1. better IDE support 10 (1.8%)
+2. better imports 7 (1.3%)
+3. better integration with other libs 3 (0.6%)
+
+
+#### Our read:
+Overall the experience is mostly positive for our users. 73% of them believe there is space for improvements - they gave a rating less than 5. Again, more documentation and training will help most users. There are also 25% of users are looking for more features.
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: `There are also 25% of users are looking` is awkward, how about `25% of users are looking`

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+
+
+<br /><br /><br />
+### Q: What would be a good cadence for Cats to release a new major version (backward incompatible with previous ones)?
+#### Results:
+![6](/img/media/2018-survey/slzrAHi.png)
+
+#### Our read: 
+The community is split on this one. There are slightly more users that prefer a longer cadence (18-36 or in sync with Scala minor version) than those who prefer a faster one (12-18 months).
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: Either`faster`/`slower` or `longer`/`shorter` here.

File: posts/2019-01-30-cats-ecosystem-community-survey-results.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+#### Our Read: 
+Breaking Cats-laws's backward compatibility might be blockers for roughly 6% of the users. 
+
+<br /><br /><br />
+### Q: Would you benefit or suffer from cats-laws and cats-testkit being updated to Scalacheck 1.14 which is binary breaking with Scalacheck 1.13?
+#### Results:
+![6](/img/media/2018-survey/xGjcmj6.png)
+#### Our Read: 
+While most people are neutral on this one, there are significantly more users (14.2%) that would benefit from a Scalacheck 1.14 upgrade than those that would suffer (1%).
```
Comment (tpolecat) [inline posts/2019-01-30-cats-ecosystem-community-survey-results.md]: s/`that`/`who`/

**Comment by kailuowang** [comment]
@tpolecat Thanks so much! Fixed.

**Comment by larsrh** [comment]
@kailuowang Are you ready to go ahead with this?

**Comment by kailuowang** [comment]
@larsrh yes, I believe so.

**Comment by larsrh** [comment]
Thanks for your hard work!

**Pull request merged**